package.json

```diff
@@ -147,5 +147,43 @@
     "@polkadot/networks": "9.0.1",
     "@polkadot/util-crypto": "9.0.1",
     "@polkadot/x-bigint": "9.0.1"
+  },
+  "overrides": {
+    "@polkadot/keyring": "9.0.1",
+    "@polkadot/api-augment": "8.0.2",
```

Comment:
These should be 8.1.1 now.

I made a PR earlier that bumped the version since 8.0.2 had a bug in pagination calls.